app.js

```diff
@@ -26,7 +27,8 @@ exports.setup = function(callback) {
   }
 
   if (is_http_thread) {
-    app.listen(CONF.app.port);
+    http = http.createServer(app);
+    http.listen(CONF.app.port);
```

Comment:
Thanks for the quick turnaround. I believe the two are identical, see https://github.com/joyent/node/blob/master/lib/http.js#L63